tests/storage/test_event_push_actions.py

```diff
@@ -175,6 +175,15 @@ def _assert_counts(noitf_count: int, highlight_count: int) -> None:
             )
             self.assertEqual(counts.threads, {})
 
+            aggregate_counts = self.get_success(
+                self.store.db_pool.runInteraction(
+                    "get-aggregate-unread-counts",
+                    self.store._get_unread_counts_by_room_for_user_txn,
+                    user_id,
+                )
+            )
+            self.assertEqual(sum(aggregate_counts.values()), noitf_count)
```

Comment:
This is an odd check for me, we really only care about the room under test?

```suggestion
			if notif_count:
	            self.assertEqual(aggregate_counts[room_id]), noitf_count)
	        else:
	        	self.assertNotIn(room_id, aggregate_counts)
```

We would do something similar in the other tests too.

(Also I just noticed that `notif` is misspelled, oops!)

Reply:
Not sure what I was thinking there! Fixed in https://github.com/matrix-org/synapse/pull/14255/commits/ed3809f90caf932ba9133007a5329a6fa0d210d8 & typo fixed in https://github.com/matrix-org/synapse/pull/14255/commits/406807965f5905d01ca4b4ddb0ed1edde8609fbf.

Reply:
Thanks for fixing the typo too! ❤️